app/src/main/java/org/yaaic/Yaaic.java

```diff
@@ -134,4 +134,25 @@ public void updateServer(Server server)
 
         return serverList;
     }
+
+
+    /**
+     * Get list of autoconnect servers
+     *
+     * @return list of autoconnect servers
+     */
+    public ArrayList<Server> getAutoconnectServersAsArrayList()
```

Comment:
It would be nice to just return the interface version: List<Server> and rename the method to getAutoConnectServers(). I now that I did the same strange thing in getServers() and this need to be changed at some point in the future too. :)